docs/site/Using-strong-error-handler.md

```diff
@@ -0,0 +1,234 @@
+---
+title: 'Using strong-error-handler'
+lang: en
+keywords: LoopBack 4, error handling
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Using-strong-error-handler.html
+---
+
+# strong-error-handler
+
+This package is an error handler for use in both development (debug) and
+production environments.
+
+In LoopBack 4, the request handling process starts with the app's
+[sequence](https://loopback.io/doc/en/lb4/Sequence.html), a simple class with
+five injected helper methods in the constructor. It is the gatekeeper of all
+requests to the app. Errors are handled by one of the Sequence actions,
+[`reject`](Sequence.md#handling-errors), which calls `strong-error-handler`
+package to send back an HTTP response describing the error.
+
+LB4 apps require 3.x versions of `strong-error-handler`.
+
+## Installation
+
+```bash
+$ npm install --save strong-error-handler
```

Comment:
For LoopBack's `@loopback/rest` package, `strong-error-handler` is already a dependency. I don't think an explicit installation is needed.

Reply:
+1